draft-ietf-pquip-pqc-engineers.md

```diff
@@ -442,45 +472,58 @@ Another important property of Diffie-Hellman is that in addition to being a NIKE
+
+The solution to binding is generally achieved at the protocol design level: do not use the KEM output shared secret directly. Even though modern KEMs such as ML-KEM produce full-entropy shared secrets, it is still advisable for binding reasons to pass it through a key derivation function (KDF) and also include all values that you wish to bind; then finally you will have a shared secret that is safe to use at the protocol level.
 
-Understanding IND-CCA2 security is essential for individuals involved in designing or implementing cryptographic systems and protocols to evaluate the strength of the algorithm, assess its suitability for specific use cases, and ensure that data confidentiality and security requirements are met. Understanding IND-CCA2 security is generally not necessary for developers migrating to using an IETF-vetted key establishment method (KEM) within a given protocol or flow. IETF specification authors should include all security concerns in the 'Security Considerations' section of the relevant RFC and not rely on implementers being deep experts in cryptographic theory.
 
 ## HPKE {#hpke}
 
-HPKE (Hybrid Public Key Encryption) {{?RFC9180}} deals with a variant of KEM which is essentially a PKE of arbitrary sized plaintexts for a recipient public key. It works with a combination of KEMs, KDFs and AEAD schemes (Authenticated Encryption with Additional Data). HPKE includes three authenticated variants, including one that authenticates possession of a pre-shared key and two optional ones that authenticate possession of a key encapsulation mechanism (KEM) private key. HPKE can be extended to support hybrid post-quantum KEM {{?I-D.westerbaan-cfrg-hpke-xyber768d00-02}}. ML-KEM does not support the static-ephemeral key exchange that allows HPKE based on DH based KEMs and its optional authenticated modes as discussed in Section 1.2 of {{?I-D.westerbaan-cfrg-hpke-xyber768d00-02}}.
+Modern cryptography has long used the notion of "hybrid encryption" where an asymmetric algorithm is used to establish a key, and then a symmetric algorithm is used for bulk content encryption.
```

Comment:
I think introducing this usage of the word "hybrid" here is going to be extremely confusing. It already has way too many meanings in the context of PQC.

Reply:
Yes, I prefer to remove the line.

Reply:
I agree that I don't want to be confusing, but I think there is a point to be made here:

1) "HPKE" already includes the word "hybrid", so there is no avoiding it.

2) I think it's worth pointing out that HPKE (9180) is not some revolutionary new thing; it's just the latest iteration of an old concept.

Reply:
The term "hybrid encryption" is used in any other RFC, please check.

Reply:
@tireddy2 I don't understand your comment. Can you phrase it a different way?

Reply:
I was looking to see if the term 'hybrid encryption' is defined in any other specification but could not find it (other than in RFC 9180). I have no objections to the proposed text.